Use java's createTempFile. Fixes #43

nickgal opened the pull request:
createTempFile generates a non-conflicting filename which avoids multiple processes trying to read/write the same file.

This fixes #43 the SIGSEGV when running multiple instances under linux.

DeJayDev commented on src/main/java/net/arikia/dev/drpc/DiscordRPC.java:
```diff
@@ -125,38 +125,34 @@ public static void discordRespond(String userId, DiscordReply reply) {
-		System.load(f.getAbsolutePath());
+			System.load(f.getAbsolutePath());
+		} catch(Exception e) {
+				e.printStackTrace();
```
Remove a tab

nickgal replied:
Not sure how that sneaked in.

DeJayDev commented:
Love it, thanks for the PR. Small gripe with whatever happened on indentation on the reviewed line.

nickgal committed:
Use java's createTempFile. Fixes #43
createTempFile generates a non-conflicting filename which avoids multiple processes trying to read/write the same file.
